spec/src/main/asciidoc/jakarta-data-introduction.asciidoc

```diff
@@ -0,0 +1,29 @@
+= Introduction
+
+Jakarta Data provides an API that makes data access easy. A
+Java developer can split the persistence from the model with several features,
+such as the ability to compose custom query methods on a Repository interface where the framework
+will implement it.
+
+== Introduction
+
+There is no doubt about the importance of data around the application. We often talk about a stateless application, the data where we delegate the application's state. If the architecture is the mind, the information is the heart.
```

Comment:
"We often talk about a stateless application, the data where we delegate the application's state."
could benefit from a minor grammar correction to:
"We often talk about a stateless application, where we delegate the application's state to the database."